app/javascript/components/searchBar/search-bar-conatiner.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import SearchBar from './search-bar';
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {filterRestaurants} from '../../actions/index';
+
+class SearchBarContainer extends React.Component{
+    constructor(props) {
+        super(props);
+
+
+        this.handleSearch = this.handleSearch.bind(this);
```

Comment:
pretty sure there is a es6 way to declare this function + the binding

https://reactjs.org/docs/faq-functions.html
the class property (second example)

Reply:
you are correct, but class properties are not fully confirmed yet according to the docs.
regarding the other options they create new function for each render, also not the best solution.

Reply:
> you are correct, but class properties are not fully confirmed yet according to the docs.
could be true , we still tend to use both styles (again with babel, shims and what not :) )

this is more of an FYI than a change request.

Reply:
👍